src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -1704,15 +1711,10 @@ private function moveNode(string $srcAbsPath, string $destAbsPath): void
 
         // TODO: Find a better way to do this
         // Calculate CAST type for CASE statement
-        switch ($this->getConnection()->getDatabasePlatform()->getName()) {
-            case 'pgsql':
-                $intType = 'integer';
-                break;
-            case 'mysql':
-                $intType = 'unsigned';
-                break;
-            default:
-                $intType = 'integer';
+
+        $intType = 'integer';
+        if ($this->getConnection()->getDatabasePlatform() instanceof MySQLPlatform) {
```

Comment:
Note that this `instanceof` check does not match MariaDB in DBAL 4. Not sure if that's intended.